ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -124,7 +124,7 @@ func (pg *ManualMixerSetupPage) Layout(gtx layout.Context) layout.Dimensions {
 											}),
 											layout.Rigid(func(gtx C) D {
 												txt := `<span style="text-color: grayText2">
-											<b>Make sure to select the corresponding accounts to the previous setup. </b> Failing to do so could damage wallet privacy.
+											<b>Make sure to select the same accounts from the previous privacy setup. </b><br>Failing to do so could compromise wallet privacy.<br> You may not select the same account for mixed and unmixed.
```

Comment:
the use of `May` from my point of view, make it feel like it a choice not to select same wallet for mixed and unmixed. I would rather use `Should`. 

```suggestion
											<b>Make sure to select the same accounts from the previous privacy setup. </b><br>Failing to do so could compromise wallet privacy.<br> You should not select the same account for mixed and unmixed.
```

Reply:
`should` makes it optional whereas `may` is mandatory. Could use `must` but I think `may` is better. Another option is `can not`.